testapi.pm

```diff
@@ -919,6 +926,12 @@ sub script_output($;$) {
         type_string("\n", terminate_with => 'EOT');
         wait_serial("$suffix-0-");
     }
+    elsif ($args{type_command}) {
+        my $cat = "cat - > /tmp/script$suffix.sh << EOF;";
```

Comment:
this approach is not working :( http://kimball.arch.suse.de/tests/279#step/wicked_before_test/28. 
Code which I am trying to execute is 
```
my $var1 =  script_output('cat /proc/stat',30,type_command => 1);
script_output("echo $var1",30,type_command => 1);
```
Any ideas what I am doing wrong ?